### PR TITLE
Numerious spell updates 

### PR DESCRIPTION
**Pull request by datdamnzotz**

### CODE REVIEW

File: src/cljc/orcpub/dnd/e5/spells.cljc [hunk 1]
```diff
@@ -1133,15 +1133,17 @@ While on the Ethereal Plane, you can see and hear the plane you originated from,
     :description "Your body becomes blurred, shifting and wavering to all who can see you. For the duration, any creature has disadvantage on attack rolls against you. An attacker is immune to this effect if it doesn't rely on sight, as with blindsight, or can see through illusions, as with truesight."
     }
    
-   {:name "Booming Blade"
+   #_{:name "Booming Blade"
```
Comment (MeanderingCode) [inline src/cljc/orcpub/dnd/e5/spells.cljc:1136]: This comments out this block, removing the spell from the list.
Is this intentional? There are several spells commented out already in this file.

File: src/cljc/orcpub/dnd/e5/spells.cljc [hunk 2]
```diff
@@ -3810,14 +3814,17 @@ A dancing creature must use all its movement to dance without leaving its space
     :description "A passage appears at a point of your choice that you can see on a wooden, plaster, or stone surface (such as a wall, a ceiling, or a floor) within range, and lasts for the duration. You choose the opening's dimensions: up to 5 feet wide, 8 feet tall, and 20 feet deep. The passage creates no instability in a structure surrounding it.
 When the opening disappears, any creatures or objects still in the passage created by the spell are safely ejected to an unoccupied space nearest to the surface on which you cast the spell."
     }
-   {
+   #_{
```
Comment (MeanderingCode) [inline src/cljc/orcpub/dnd/e5/spells.cljc:3817]: Again removing a spell.

File: src/cljc/orcpub/dnd/e5/spells.cljc [hunk 3]
```diff
@@ -2755,15 +2758,17 @@ When the grease appears, each creature standing in its area must succeed on a De
 • Any reduction to one of the target's ability scores
 • One effect reducing the target's hit point maximum"
     }
-   {:name "Green-Flame Blade"
+   #_{:name "Green-Flame Blade"
```
Comment (MeanderingCode) [inline src/cljc/orcpub/dnd/e5/spells.cljc:2761]: Again removing a spell.

**Review by MeanderingCode — COMMENTED** [review]
This PR updates a few spells, and removes a few by commenting them out.

Is it the intention to remove those spells?

File: src/cljc/orcpub/dnd/e5/spells.cljc (hunk 1 above)
Comment (datdamnzotz) [reply src/cljc/orcpub/dnd/e5/spells.cljc:1136]: Yes, its not in SDR
https://github.com/larrychristensen/orcpub/issues/192

Comment (MeanderingCode) [reply src/cljc/orcpub/dnd/e5/spells.cljc:1136]: Makes sense.
I'll merge.

**Review by MeanderingCode — APPROVED** [review]

**Pull request merged**